wiki/Community/Project_Loved/en.md

```diff
@@ -32,11 +32,13 @@ Although many [Pending and Work-in-progress](/wiki/Beatmap/Category#work-in-prog
 
 Beatmaps entering the Loved category are required to follow some additional rule sets:
 
-- [The Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata)
+- [Partially the Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata)
```

Comment:
I don't know how to read this section.

Reply:
The following sub-bulletpoints were meant to underline which sections of the mentioned portion of RC the beatmap being moved to Loved should abide by. If it's not well-enough formed, I can try phrasing it differently.